python/ray/_private/services.py

```diff
@@ -1929,3 +1929,22 @@ def start_ray_client_server(redis_address,
         stderr_file=stderr_file,
         fate_share=fate_share)
     return process_info
+
+
+def get_object_locations(obj_refs, timeout_ms=-1):
```

Comment:
Can you also explain the input parameters? You can probably follow our docstring pattern here; https://github.com/ray-project/ray/blob/f14f197d42c5dcd7c24bfd68a919c27ed515da29/python/ray/worker.py#L1541